docs/en/reference/schema-representation.rst

```diff
@@ -45,19 +48,11 @@ use the ``Comparator`` class to get instances of ``SchemaDiff``,
 foreign key, sequence and index changes.
 
 .. code-block:: php
-
     <?php
+    /*...*/
     $schemaManager = $connection->createSchemaManager();
     $comparator = $schemaManager->createComparator();
-    $schemaDiff = $comparator->compare($fromSchema, $toSchema);
-
-    $queries = $schemaDiff->toSql($myPlatform); // queries to get from one to another schema.
```

Comment:
Why remove this?

Reply:
`compare` replaced to `compareSchemas`, `toSql`/`toSaveSql` methods removed

Reply:
The purpose of this code example is to show how to migrate from one schema to another. Now it doesn't show that.

> `toSql`/`toSaveSql` methods removed

See the upgrade notes: https://github.com/doctrine/dbal/blob/750d846074fe920b65dcfca3c9bdcea45fa78a97/UPGRADE.md?plain=1#L164-L165

Reply:
I just fix outdated information. If you want extend example, you can do it.

Reply:
@WinterSilence You changed the purpose of this code example. It is supposed to stay how to migrate from one schema to another without the deprecated methods.

Reply:
@SenseException @morozov what solution do you suggest? save outdated solution and don't do anything?